src/ticketmap/index.tsx

```diff
@@ -151,8 +152,9 @@ export default class TicketMap extends Component<Props & DefaultProps, State> {
 
   setupMap() {
     const mapSvg = this.mapRootRef.querySelector('svg') as SVGSVGElement
-    mapSvg.style.width = '100%'
-    mapSvg.style.height = '100%'
+    const [x, y, width, height] = mapSvg.getAttribute('viewBox').split(' ')
```

Comment:
Do all of our maps have `viewBox` attributes?

Reply:
Yes they do